getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -133,15 +133,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 22,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Project name: getting-started-tutorial-iguazio\n",
-      "Artifacts path: /v3io/projects/{{run.project}}/artifacts\n"
+      "Project name: getting-started-tutorial-edmond\n",
```

Comment:
Probably better to run the code with user iguazio to keep the text output the same

Reply:
OK